mucgpt-frontend/src/components/ChatToolSelector/ChatToolSelector.module.css

```
@@ -0,0 +1,642 @@
+}
+
+.accordionGroupHeader:hover {
+    background-color: var(--colorNeutralBackground2Hover);
+}
+
+.accordionHeader {
+    flex: 1;
+    min-width: 0;
+}
+
+.accordionHeader :global(.fui-AccordionHeader__button) {
```

Stylelint (17.7.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)